### PR TITLE
[AVRO-1837] adding support for date, time and timestamp logicaltype type support

### PR DESCRIPTION
**Pull request by abrarsheikh**

### CODE REVIEW

File: lang/py/src/avro/io.py [hunk 1]
```diff
@@ -38,8 +38,11 @@
 """
 import struct
 from avro import schema
+from avro import constants
+from avro.lib import timezones
```
Comment (charlesdurand) [inline lang/py/src/avro/io.py]: @abrarsheikh you probably forgot to commit this file (this import fails)
Using `pytz` and `pytz.utc` works fine.

**Comment by idealopamp** [comment]
Would love to see this get merged in.  Anything that the community can help out with on pushing this forward?

**Comment by abrarsheikh** [comment]
@charlesdurand i am will get that fixed right away. I was reluctant use `pytz` because that introduces another dependency to the project.

@idealopamp++

sorry for the delayed response on this

**Comment by slepkin** [comment]
Anything else blocking this? Eager for it to be accepted.

**Comment by k4rt33k** [comment]
Agree with @slepkin.

**Comment by busbey** [comment]
Reviews always help get things merged faster.

Thanks for all the work on this so far!

Could someone go through checking interoperability with the logical type implementation in another language (I'd guess Java is probably the only option)?

**Comment by derekenos** [comment]
:+1: for getting this merged.
I did try to check interop with the java implementation but java made me lose my will to continue.

**Comment by derekenos** [comment]
That's great news, @abrarsheikh. Thanks for the update.

**Comment by dharamsk** [comment]
This would be super useful for us. Any idea when it might be part of an avro release?

**Comment by Fokko** [comment]
@abrarsheikh Would be interesting to get this in, can you rebase against master?

**Pull request closed without merging**